src/daemon/rpmostreed-sysroot.cxx

```diff
@@ -549,11 +549,21 @@ rpmostreed_sysroot_init (RpmostreedSysroot *self)
   /* Only use polkit when running as root on system bus; self-tests don't need it */
   if (!self->on_session_bus)
     {
-      g_autoptr (GError) local_error = NULL;
-      self->authority = polkit_authority_get_sync (NULL, &local_error);
-      if (self->authority == NULL)
+      // See https://github.com/coreos/rpm-ostree/issues/4323 and
+      // https://github.com/coreos/fedora-coreos-tracker/issues/334 and
+      // https://lists.freedesktop.org/archives/dbus/2023-March/018252.html Basically, starting
+      // polkit can time out on slow systems; we want to just keep retrying so that we hit the
+      // systemd unit activation timeout (90s); crucially whatever it's configured to be, so that
+      // slow systems can bump that timeout.
+      while (true)
```

Comment:
Shouldn't this be e.g. `while (self->authority == NULL)`?

Reply:
Hah yes, I tested an earlier version of this patch which had a `for (guint i = 0; i < 5; i++)` but later changed it to be `while (true)` and apparently failed to sync over my testing binary again...